Gemfile

```diff
@@ -47,7 +47,6 @@ gem "tzinfo-data"
 gem "activerecord-import"
 gem "govuk-components", ">= 1.0.0"
 gem "govuk_design_system_formbuilder", "~> 2.1", ">= 2.1.5"
-gem "govuk_publishing_components", ">= 23.11.0"
```

Comment:
Sweet. Do we need `sass-rails`? I think I added it when I added govspeak styling, on https://github.com/DFE-Digital/early-careers-framework/pull/42